assets/logic.js

```diff
@@ -1,4 +1,29 @@
 $('.jsToggle').on('click', function(){
 	var target = $(this).data('toggle');
 	$(target).toggleClass('active bounceInDown');
+});
+
+$(document).ready(function(){
+	function getRandom(min, max) {
+		return Math.floor(Math.random() * (max - min + 1)) + min;
+	}			
+	var canvas = document.getElementById('canvasContainer');
```

Comment:
Is it worth breaking all of this canvas stuff into a function for nicer readability?

function starify???

starify();

That way you can run it on pages that have canvasContainer?